src/test/kotlin/logic/use_cases/log/GetAllLogsUseCaseTest.kt

```diff
@@ -48,7 +48,7 @@ class GetAllLogsUseCaseTest{
     }
 
     @Test
-    fun `should return list of all logs when repository returns logs`() {
+    fun `returns all logs when repository has logs`() {
```

Comment:
You should follow the same naming pattern as the entire testing functions

```suggestion
    fun `should return all logs when repository has logs`() {
```